README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
let's remove changes in docs?

Reply:
oh i just enable `end-of-file-fixer` under pre-commit. Should we disable that?

Reply:
Probably, doesn't it mess up the format in rst or md files?

Reply:
i don't think so, but we can chat more